src/libOpenImageIO/exif.cpp

```diff
@@ -783,7 +783,7 @@ read_exif_tag(ImageSpec& spec, const TIFFDirEntry* dirp, cspan<uint8_t> buf,
     if (dir.tdir_tag == TIFFTAG_EXIFIFD || dir.tdir_tag == TIFFTAG_GPSIFD) {
         // Special case: It's a pointer to a private EXIF directory.
         // Handle the whole thing recursively.
-        unsigned int offset = dirp->tdir_offset;  // int stored in offset itself
+        int offset = dirp->tdir_offset;  // int stored in offset itself
```

Comment:
That doesn't make sense. If you look in tiffutils.h, `tdir_offset` is clearly unsigned.

What version of libtiff are you building against?

Reply:
I'm using tiff version 4.1.0